packages/core/src/components/DotPagination/DotPagination.tsx

```diff
@@ -85,22 +83,24 @@ export const HvDotPagination = ({
 }: HvDotPaginationProps) => {
   const range = (n: number) => Array.from(Array(n), (_, i) => i);
 
+  const { classes, cx } = useClasses(classesProp);
+
   const icons = getSelectorIcons(unselectedIcon, selectedIcon, classes);
 
   return (
-    <StyledRadioGroup
-      className={clsx(className, classes?.root, dotPaginationClasses.root)}
+    <HvRadioGroup
+      className={cx(className, classes.root)}
```

Comment:
`className` should come after, as it's a user-defined and should override our styles
```suggestion
      className={cx(classes.root, className)}
```